Geometry/HGCalGeometry/test/python/hgcGeomAnalyzer_cfg.py

```
@@ -0,0 +1,37 @@
+#run with: cmsRun hgcGeomAnalyzer_cfg.py geom=v10
+
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+from Configuration.StandardSequences.Eras import eras
+
+options = VarParsing()
+options.register ("geom", "",  VarParsing.multiplicity.singleton, VarParsing.varType.string)
+options.parseArguments()
+
+fileName = "geom_output_"+options.geom
+
+process = cms.Process("demo",eras.Phase2C8)
+
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+if options.geom == 'v10':
+    process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+elif options.geom == 'v11':
+    process.load('Configuration.Geometry.GeometryExtended2026D46Reco_cff')
+else:
+    print 'UNKNOWN GEOMETRY!'
```

better to raise an exception here